boards/BMS/Test/Src/Test_VoltageSense.cpp

```diff
@@ -1,18 +0,0 @@
-#include "Test_Bms.h"
-
-extern "C"
-{
-#include "Io_VoltageSense.h"
-}
-
-TEST(VoltageSenseTest, tractive_system_voltage_calculation)
```

Comment:
Would it be beneficial to add a test for voltage sense somewhere else now that it's in App code?

Reply:
In theory yes, but all it would test is the resistor divider values
Since we'll have to test this on hardware anyway and we don't test IO code anywhere else I think it's fine